xpk.py

```diff
@@ -3120,7 +3099,7 @@ def get_capacity_arguments_from_capacity_type(
       capacity_args = '--spot'
     case CapacityType.RESERVATION:
       capacity_args = (
-          f'--reservation-affinity=specific --reservation={args.reservation}'
+        f'--reservation-affinity=specific --reservation={args.reservation} --placement-policy={args.reservation}'
```

Comment:
Is this needed in TPUs / CPUs? I see the build test failing with TPUs, I assume related to this.